### DISCUSSION TITLE
Batch Size and Energy + Force Training

### DISCUSSION THREAD

**Discussion by rschireman** [mir-group/allegro#81] [Q&A] [answered]
Hi all, thanks for maintaining this code. I'm attempting to train a model for just ice and I noted that batch sizes should be really small for force training (1-5). I've also confirmed this, but is there a known reason why?

**Comment by rschireman** [comment]
for example, with the same model parameters, I'll notice a significant difference between `batch_size=5` vs `batch_size=2`, where 2 is the better option

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @rschireman ,

I'm not aware of a careful study of the reasons for this, but your observation is consistent with everything we've observed.  Larger batch sizes seem to work best for very diverse training sets.

**Reply by rschireman** [reply]
thanks!
